### ISSUE TITLE
Feature: adding a recursive option to fetch `.bib` files ?

### ISSUE THREAD

**Issue by Thomzoy** [shyamd/mkdocs-bibtex#180]
Hi !

First of all thanks for this great repo !
I forked it since in my project, I  use multiple `.bib` files scattered across the `docs` folder. As such, I added a `recursive` option with a simple change when fetching the `.bib` files :

```python
if self.config["recursive"]:
    bibfiles.extend(Path(self.config["bib_dir"]).rglob("*.bib"))
else:
    bibfiles.extend(Path(self.config["bib_dir"]).glob("*.bib"))
```

Would you be interested in a PR for this ?

**Comment by shyamd** [comment]
Would definitely appreciate the PR. No need to add `recursive` option. Just make it the default since that seems like a sensible expectation.

**Comment by shyamd** [comment]
This got released in `v2.8.15`
Thanks!
